### ISSUE TITLE
Breadcrumbs and urls of news items should not contain publisher

### ISSUE THREAD

**Issue by nicholasgkdto** [AusDTO/gov-au-beta#265]
The format for url is 
`platform/department-of-communications-and-the-arts/news/2016-04-29-nbn-satellite-service-opens-for-business`
and should be 
`platform/news/2016-04-29-nbn-satellite-service-opens-for-business`

For breadcrumb it should NOT be 
`Home > Department of Communications and the Arts > News > nbn satellite service opens for business`
and should be 
`Home > News > nbn satellite service opens for business`

**Comment by ltankey** [comment]
@nicholasgkdto this makes same-named news items published on the same date impossible, as the namespace would clash for URLs.

A typical use-case might be a generic news title ("EOFY", "End of caretaker" etc.) that multiple areas might publish on the same date. As we would need to provide a unique URL to each of these, this requires the News Articles hang off a news section from the publishing area.

Do you have any thoughts with your suggestions above regarding this particular issue?

**Comment by maadonna** [comment]
So here's what I thought...

A section will have a news listing page:
- URL is /name-of-section/news
- breadcrumb is Home > Name of section > News
- there is an item in the navigation bar called News

When you click on a story, you actually leave the section, and end up in the news section (though you and I might use 'section' slightly differently):
- URL is news/name-of-story-and-date
- Breadcrumb is Home > News > Name of story
- The news story has no navigation bar

Sorry! I really wish we'd stopped and mapped this out instead of working off rough prototypes.

**Comment by ltankey** [comment]
Ok thanks @maadonna - a better question might be how urgent this requirement is? Is this something we expect to be critical over the next few weeks?

**Comment by maadonna** [comment]
Not urgent, but something we still should aim to get right. The breadcrumb is a little more important than the URL as it's more visible to people...

**Comment by ltankey** [comment]
@maadonna does this also mean that whilst on /section/news, the breadcrumbs would only show Home > News ?

**Comment by maadonna** [comment]
Yes :) (as long as you mean 'when I'm looking at the news listing' and not 'when I'm looking at a news story')

**Comment by micapam** [comment]
@maadonna As a user, if I'm browsing e.g. comms & arts news, and I click on a story, I wouldn't expect to suddenly leave the department. I'd feel quite at-sea to suddenly be in gov.au main news section. 

On the other hand, If I'm browsing the main news site, and I click on a story on which the publisher is clearly shown (in the listings), it would make more sense for me to be taken to that publisher section.

That might just be me though. Should we get user research to back this up before making these changes...?

**Comment by ltankey** [comment]
@micapam we're going to discuss this at tech strategy before making any changes to implementation, for the very reasons you've described but also namespace clash.

However I'll bundle in some cheap breadcrumbs issues because they're trivial to make.

@maadonna so it's ok for the URL in this instance to differ from the breadcrumbs?

**Comment by maadonna** [comment]
That's absolutely what we're doing. And believe it or not, people will be fine, and have been in testing so far. Most won't have a strong concept of whether they are in a department section a topic section etc. They are still in 'the government website' they were in, and will hardly notice. This is an absolutely fundamental concept in being able to share content - by authoring it once in a central place, we can display it in more than one section (e.g. 2 topics and a department) to allow users to find and navigate it in more than one way. This is absolutely central to the gov.au IA.

**Comment by maadonna** [comment]
@ltankey yes, OK for the URL to differ from the breadcrumb in the short term. But let's work through it again and get it all aligned longer term

**Comment by ltankey** [comment]
The breadcrumbs component of this is currently being reviewed and will be delivered as part of our next release. The URL component has been deprioritised and will be discussed at our next tech strategy meet-up this week. 

I'll leave this ticket open until the URLs have been resolved too.

**Comment by maadonna** [comment]
I'm going to try to listen in to the tech strategy meeting this week. I won't contribute or de-rail anything - it just helps me catch any of these things earlier if I know more about your bigger picture stuff...

**Comment by ltankey** [comment]
Breadcrumbs resolved in v0.7.0

I'll leave this open for the URL discussion.

**Comment by Alex-DTO** [comment]
I'm just catching up on everything on this, apologies I'm late to the party.

I actually agree with @micapam - if I came from a Topic, looked at their News page, and selected an item, I would expect my journey to remain within that topic. I would also expect to be able to easily get to global news but I think the use case for this is unlikely....

For now let's leave this as we have built it - aim for minimal disruption. I would like to understand this a little better next week and get a better idea of the testing we went through. I'll add this to my backlog for the time being

**Comment by maadonna** [comment]
You have to remember that the news story lives centrally. That way we can make it navigable via more than one topic, department, minister etc. It must have one URL. If it were to stay in the topic journey, it would need a topic URL. If it was also visible via a minister and stayed in that journey, it would have to have a minister URL too. We absolutely can't do that. Although we will design topics so that mostly people stay within them, there are going to be plenty of times where people click links that take them to another topic, an agency, a minister. They aren't going to get 'lost'. They are still within the big government website and can use the back button.

So far, I haven't seen any indication in testing that there was any kind of problem. People clicked on links, read things, chatted about their experience. No-one went 'OMG - I just went somewhere entirely different'.

As I said before, this 'shared content' concept is absolutely core to the IA for gov.au, and is one of the benefits of pulling everything into one website. Otherwise we could have just left the organisations to have their own URLs for their corporate sites; and could have built topic websites. The ability to author content once and have it displayed and navigable from more than one place is a huge benefit over the current approach.

**Comment by micapam** [comment]
Maybe (ironically!) because of our team structures in the development of govau, we may have lost sight of that - the 'one big govau website' aspect - so thanks for reminding us @maadonna !

**Comment by micapam** [comment]
(FWIW I no longer agree with my earlier comments...)

**Comment by Alex-DTO** [comment]
lol!!! Donna, let's talk next week.... I still kiiiiiinda cling to my comments - a news item can and must be created in one way and 'belong' to News, and appear elsewhere according to how it is tagged. The experience that an individual gets on that article when they come through a Section --> Section News should be malleable to what users say is best (pending more testing later if we need to) - i.e. the very same news article could appear in a number of places on the site, and middle of the URL differs depending on where it was surfaced.

That way if someone shares this, their friends will see the news, and gain the content within which the original user first read the item. Instead of being on a global page of news without this...

**Comment by maadonna** [comment]
If the middle of the URL differs, then we have 2 (or more - up to 4 if it relates to 2 topics and 2 organisations) URLs for one thing. Bad idea generally, bad for SEO, google hates it :)

This is why we have built in links so that people can navigate to the section/s it relates to, though I'm not sure they're as clear as they need to be - when I looked at a news story earlier, I couldn't see how to navigate to the relevant sections. I do need to look at that again and find out what happened.

**Comment by Alex-DTO** [comment]
Yeah I think that's a defect and I've raised it separately. Let's take a look next week

**Comment by maadonna** [comment]
Just recording a discussion with @ltankey & @tessereth. If we have two news stories with the same title on the same day, it's not going to be a particularly good experience for someone looking at the global news listing. Discussed that we could let the author know that there is already a news story with the same name, suggest they change it (but not make that mandatory); and if it is still the same, append a number to the end of the URL.
